app/services/TrustService.scala

```diff
@@ -30,8 +30,8 @@ class TrustServiceImpl @Inject()(connector: TrustsConnector) extends TrustServic
   override def getAssets(identifier: String)(implicit hc:HeaderCarrier, ec:ExecutionContext): Future[Assets] =
     connector.getAssets(identifier)
 
-  override def getMonetaryAsset(identifier: String)(implicit hc: HeaderCarrier, ex: ExecutionContext): Future[Option[AssetMonetaryAmount]] =
-    getAssets(identifier).map(_.monetary.headOption)
+  override def getMonetaryAsset(identifier: String, index: Int)(implicit hc: HeaderCarrier, ex: ExecutionContext): Future[AssetMonetaryAmount] =
```

Comment:
I'm trying to understand why this was originally without an index. 
Is it because we can only have one money asset? 
If that is the case do we need the index?